qcfractal/interface/client.py

```diff
@@ -466,16 +466,16 @@ def add_collection(self, collection: Dict[str, Any], overwrite: bool=False,
 ### Results section
 
     def query_results(self,
-                      id: QueryObjectId=None,
-                      task_id: QueryObjectId=None,
-                      program: QueryStr=None,
-                      molecule: QueryObjectId=None,
-                      driver: QueryStr=None,
-                      method: QueryStr=None,
-                      basis: QueryStr=None,
-                      keywords: QueryObjectId=None,
-                      status: QueryStr="COMPLETE",
-                      projection: QueryProjection=None,
+                      id: 'QueryObjectId'=None,
```

Comment:
Curiosity question, how much time do all these delayed type loads save?

Reply:
It is to improve the documentation. The objects were coming up as `QCFractal.interface.model.QueryObjectId` in the documentation which is bad.